support/README.md

```diff
@@ -0,0 +1,17 @@
+---
+sort: 4
+---
+
+# Training and Support
+
+## Training
+
+Are you interested in receiving training from a solids4foam specialist? We offer a range of training packages. The format and content can be tailored to your specific need. Whether online or in-person, for beginners or advanced, we can accommodate your requirements.
+
+Take a look at our public [training material](training-material). For more information, contact: contact@solids4foam.com
+
+## Consulting and Support
+
+Our consulting team provide solutions to a wide range of computational problems. These include developing, implementing and verifying models and methods for **computational solid mechanics**, **fluid-solid interactions**, and **computational fluid dynamics**. In addition to providing support, the solids4foam team is available to join collaborative research projects and funding proposals. For general enquiries, reach out at: contact@solids4foam.com
```

Comment:
@philipcardiff I like the concise nature of this compared to the previous version. What are thoughts on the following format changes:

```suggestion
Our consulting team provide solutions to a wide range of computational problems. These include developing, implementing and verifying models and methods for:

- **Computational solid mechanics**
- **Fluid-solid interactions** 
- **Computational fluid dynamics** 

In addition to providing support, the solids4foam team is available to join collaborative research projects and funding proposals. For general enquiries, reach out at: contact@solids4foam.com
```

I think this might draw the reader's attention and make it easier to read instead of one block of text.

It's a minor change either way, and it will probably have to be changed as more information gets added in the future.

Reply:
OK, sure.